go/ct/st/code_test.go

```diff
@@ -140,6 +140,57 @@ func TestCode_Printer(t *testing.T) {
 	}
 }
 
+func TestCode_OpCodesToString(t *testing.T) {
+	tests := map[string]struct {
+		code   *Code
+		start  int
+		length int
+		want   string
+	}{
+
+		"empty": {
+			code:   NewCode([]byte{}),
+			start:  0,
+			length: 4,
+			want:   "len(0)",
+		},
+		"complete": {
+			code:   NewCode([]byte{byte(ADD), byte(PUSH1), 0x0C, byte(PUSH2)}),
+			start:  0,
+			length: 4,
+			want:   "len(4); ADD; PUSH1; op(0C); PUSH2",
```

Comment:
nit: if you replace the `0x0C` with e.g. `0x01` the output would read `ADD, PUSH1, ADD, PUSH2` ...

Reply:
That was my first idea: it broke other tests for the opcode. String function. I did not want to risk breaking something else by changing it.

Reply:
done